MapboxNavigation/ErrorCodes.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/**
+ Enum used for indicating the type of error occured while speaking an instruction.
+ */
+public enum MapboxNavigationError: Int {
```

Comment:
Rename this enumeration to `MBErrorCode` for consistency with `MGLErrorCode` in the map SDK.